compair/static/modules/home/home-partial.html

```diff
@@ -18,12 +18,30 @@
 
     <div class="row">
 
-        <h2 class="col-sm-8">Your courses</h2>
+        <h2 class="col-sm-6">Your courses</h2>
 
-        <form class="col-sm-4 search-courses text-right" role="search" ng-show="canAddCourse && (courses.length || courseFilters.search)">
-            <div class="form-group">
+        <form class="col-sm-6 search-courses text-right" role="search" ng-show="!canManageUsers && canAddCourse && (courses.length || courseFilters.search)">
+            <input class="form-control" type="text" name="search" placeholder="Filter courses by keyword"
+                ng-model="courseFilters.search">
+        </form>
+
+        <form class="col-sm-6 search-courses text-right" role="search" ng-show="canManageUsers && canAddCourse && (courses.length || courseFilters.search || courseFilters.includeSandbox!='')" style="padding-right:0;">
```

Comment:
The conditions on this need to be split. I believe it should be 
`canAddCourse && (courses.length || courseFilters.search || courseFilters.includeSandbox!=null)`

Reply:
@andrew-gardener what's the purpose of splitting this condition?

Reply:
instructors (having `canAddCourse` but not `canManageUsers`) can no longer search

Reply:
There's another form element for instructors just before this form element.